pkgs/build-support/libredirect/default.nix

```diff
@@ -1,4 +1,10 @@
-{ stdenv, lib, coreutils }:
+{ stdenv, lib, coreutils, cairo
```

Comment:
Maybe some smaller library would be a better choice?

Reply:
I feel it's appropriate to test against it.